README.md

```diff
@@ -66,7 +66,7 @@ Supported Distributions
 -----------------------
 
 * RHEL/CentOS 7.4+
-* Fedora 26+
+* Fedora 34+
```

Comment:
Why version 34?

Reply:
[Based on this document](https://docs.fedoraproject.org/en-US/quick-docs/fedora-and-red-hat-enterprise-linux/index.html) this is the base of CentOS Stream 9/RHEL 9.

Reply:
As discussed offline, I'm changing Fedora minimum supported version to 40.